### ISSUE TITLE
Recurrent Neural Nets

### ISSUE THREAD

**Issue by rahul-ahuja** [udacity/dlnd-issue-reports#225]
Some students have raised why should `embed_dim = rnn_size` as per in the project 3. https://discussions.udacity.com/t/why-should-embed-dim-rnn-size/232661
Both are different hyper-parameters?

Also, in sentiment rnn lessons, it's been told to filter out the review with 0 length.
But then the feature matrix has been initialized using len(reviews) which is unfiltered.
https://discussions.udacity.com/t/sentiment-prediction-rnn-solution-bug/233051

**Comment by mcleonard** [comment]
The second part has been addressed already. @Brok-Bucholtz for the first part?

**Comment by rahul-ahuja** [comment]
thanks Mat! Let me know about that.

Get Outlook for Android<https://aka.ms/ghei36>


________________________________

**Comment by Brok-Bucholtz** [comment]
Thanks for letting us know.  We have added a hyperparameter for dimension size.
